### ISSUE TITLE
Feature request: add the charms' vanilla locations to the rando pool

### ISSUE THREAD

**Issue by BadMagic100** [dpinela/Transcendence#5]
Probably contingent on #3 , and having a setup for logic (though adding logic for new locations is probably easier than modifying existing logic)

Many of the charm locations are unique and interesting  so it would be nice to have those locations randomized (though I'm personally not a huge fan of the ones right next to other checks). Additionally, doing this would reduce shop bloat

**Comment by dpinela** [comment]
I think that would make more sense as its own rando add-on. Not only because it's outside the purpose of this mod, but also because then it would be free to add a lot more locations besides the ones featured here.
